combine_features.py

```diff
@@ -32,6 +32,9 @@
 parser.add_argument("-n", "--new_feature_name", dest="new_feature_name",
                     help="The new name of the combined feature",
                     metavar="NAME", required=True)
+parser.add_argument("-g", "--groupName", dest="groupName",
+                    help="Feature group name.",
+                    metavar="GROUPNAME", default="enterGroupName")
```

Comment:
Instead of `"enterGroupName"` I think we want something like `"unspecifiedGroupName"` and this should be a default throughout the changes to this repo.